### PR TITLE
build: add prettier instead of tslint

### PR DESCRIPTION
**Pull request by bolmaster2**
With the argument that tslint is deprecated and eslint should be used instead for linting. 

...and then add prettier for formatting which IMO is the industry standard _and_ very handy to not care about code formatting and it's taking the decision for us

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -9,6 +9,23 @@ on:
...
 jobs:
+  lint:
+    name: Linting with prettier
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          persist-credentials: false
```
Comment (loffek) [inline .github/workflows/ci.yml:20]: This is only needed for commit lint to work, so no need to add it here (but see next comment)

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -9,6 +9,23 @@ on:
...
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14.x'
+      - name: Install dependencies
+        run: yarn --frozen-lockfile --non-interactive --silent
+      - name: Run prettier
+        run: npm run prettier-check
```
[the remark this answers is not in the document]
Comment (loffek) [reply .github/workflows/ci.yml:28]: And remove lines:

https://github.com/getdreams/dreams-web-sdk/pull/22/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR37-R39

https://github.com/getdreams/dreams-web-sdk/pull/22/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR54-R55y

**Review by loffek — CHANGES_REQUESTED** [review]
Sounds good to me!

File: .github/workflows/ci.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (bolmaster2) [reply .github/workflows/ci.yml:28]: I just realized it's not needed in the ci at all, or do it have any purpose here when we run it on pre-commit?

[the remark this answers is not in the document]
Comment (loffek) [reply .github/workflows/ci.yml:28]: The problem is that it is optional to install lefthook, so we need to *verify* in the CI that everything is formatted correctly:

- pre-commit: run it in write mode
- in CI: run it in verify / check mode

**Comment by bolmaster2** [comment]
I restarted fresh in a new PR here instead: https://github.com/dreamstechnology/dreams-web-sdk/pull/31

**Pull request closed without merging**